textsemantics/textrank/keywords.py

```diff
@@ -0,0 +1,534 @@
+"""
+Module was removed form the gensim - this is a fixed copy
+
+This module contains functions to find keywords of the text and building graph
+on tokens from text.
+
+Examples
+--------
+Extract keywords from text
+
+.. sourcecode:: pycon
+
+    >>> from textsemantics.textrank import keywords
+    >>> text = '''Challenges in natural language processing frequently involve
+    ... speech recognition, natural language understanding, natural language
+    ... generation (frequently from formal, machine-readable logical forms),
+    ... connecting language and machine perception, dialog systems, or some
+    ... combination thereof.'''
+    >>> keywords(text).split('\\n')
+    [u'natural language', u'machine', u'frequently']
```

Comment:
I get normal strings, not Unicode. Is it important?

Reply:
It is ok. In python3 strings are Unicode without the `u'` prefix. I think it is there because of Python2.